challenges/objects-arrays.js

```diff
@@ -89,32 +113,42 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
-console.log(animalNames);
+let names = zooAnimals.forEach(function(i){
+  zooAnimals.push(i.scientific_name + i.animal_name);
+});
+console.log(zooAnimals);
 
 /* Request 2: .map()    
 
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
-
-const lowerCase = [];
+const lowerCase = animalNames.map((zooAnimal) => {
+  return scientific_name.toLowerCase();
+});
 console.log(lowerCase); 
 
+
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const largerPopulation = [];
-console.log(largerPopulation);
+// const largerPopulation = zooAnimals.filter((pop) => {
+//   return pop.zooAnimals === 5;
+// });
+// console.log(largerPopulation);
 
-/* Request 4: .reduce() 
+// /* Request 4: .reduce() 
 
-The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
+// The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
 
-*/
-const populationTotal = 0;
-console.log(populationTotal);
+// */
+// const populationTotal = zooAnimals.reduce(zooAnimals, pop) => {
+//   return zooAnimals + pop.zooAnimals;
```

Comment:
Very very close here! This is the same issue with the last problem - does each element in your array have a zooAnimals property? If not, what property does it have that would work here?